### ISSUE TITLE
Error searching for any symbol

### ISSUE THREAD

**Issue by diamant-x** [lildude/ha-addon-ghostfolio#7]
Hi,

Thanks for the time to deploy a HA addon for this. I see it's quite recent so not sure if it's ready for showtime or not.
Anyways I proceeded to install your postgre and ghostfolio addons. It runs and I can navigate around. However, when I attempt to add any symbol to start testing with data it fails to look it up. Even the MSFT symbol which is [the example used](https://github.com/ghostfolio/ghostfolio/blob/main/test/import/ok.csv) in the official docs fails and can't proceed.
I understand YAHOO source should be available by default, but let me know if I need to explicitly set it up as a platform, and if so, what to fill there?
Thanks

![image](https://github.com/lildude/ha-addon-ghostfolio/assets/7346338/6546b4b1-e1d0-40da-902b-0dce39452ad5)

![image](https://github.com/lildude/ha-addon-ghostfolio/assets/7346338/a9a99ec0-5899-46c6-870c-b2cca111a0e5)

**Comment by lildude** [comment]
Hey there. Yeah, it's a little early for prime time but it was working recently but now I see this is failing for me too. In the log I've getting errors like:

```
[Nest] 305  - 04/16/2024, 1:40:51 PM   ERROR [YahooFinanceService] Error: Invalid Crumb. For Developers - https://bit.ly/yahoo-finance-api-feedback
[Nest] 305  - 04/16/2024, 1:40:51 PM    WARN [YahooFinanceService] Fallback to yahooFinance.quoteSummary()
[Nest] 305  - 04/16/2024, 1:40:51 PM   ERROR [YahooFinanceService] Could not get quote summary for USDGBP=X
[Nest] 305  - 04/16/2024, 1:40:51 PM   ERROR [YahooFinanceService] Could not get quote summary for USDUSX=X
[Nest] 305  - 04/16/2024, 1:40:51 PM   ERROR [YahooFinanceService] Could not get quote summary for USDGBp=X
[Nest] 305  - 04/16/2024, 1:40:51 PM   ERROR [YahooFinanceService] Error: Invalid Crumb. For Developers - https://bit.ly/yahoo-finance-api-feedback
```

So either something has changed on the Yahoo or Ghostfolio side or I've fudged something up without realising it when trying to get Ingres working 😬 

My devcontainer stopped behaving on Sunday (I think something changed in the upstream addon devcontainer) and I've not had the chance to dig into why. I'll dig into this as soon as I can get a working devcontainer again.

**Comment by lildude** [comment]
I finally worked out the cause of my devcontainer not working and have now looked into this further and this is affecting the upstream self-hosted version too. I can't resolve any normal stocks, like MSFT, and get exactly the same error.

They have https://github.com/ghostfolio/ghostfolio/issues/3276 which is showing as related to https://github.com/gadicc/node-yahoo-finance2/issues/764 which was resolved about 3 hours ago, so I'd expect this to be fixed in the next version of the upstream project which I'll then pull in.

**Comment by lildude** [comment]
Upstream PR pulling in the fix: https://github.com/ghostfolio/ghostfolio/pull/3286

**Comment by diamant-x** [comment]
What an unfortunate coincidence that this fails the day i attempt to try it... Thanks for finding out and keeping up to date. I'll look forward for the upstream fix.

**Comment by lildude** [comment]
I've now released v1.2.0 which pulls in Ghostfolio 2.73.0 and I've confirmed I can now lookup symbols again. Closing as resolved.

**Comment by diamant-x** [comment]
I've pulled the latest and confirm search works again thanks. I'll be testing it further once I have some time, thanks!
